pyfar/signals/deterministic.py

```diff
@@ -194,10 +212,112 @@ def linear_sweep_time(n_samples, frequency_range, n_fade_out=90, amplitude=1,
     return signal
 
 
+def linear_sweep_freq(
+        n_samples, frequency_range, start_margin, stop_margin, fade_in=None,
+        fade_out=None, bandpass_order=8, sampling_rate=44100,
```

Comment:
```suggestion
        n_samples, frequency_range, start_margin, stop_margin, fade_in=0,
        fade_out=0, bandpass_order=8, sampling_rate=44100,
```
Is this type conversion required here? The docstring states that the type is 'int'
Also, this parameter is inconsistent with the time domain sweep generation function, which uses n_fade_in.

Reply:
Thats true - renamed to `n_fade_*` and used `0` as the default